app/src/main/java/org/wikipedia/descriptions/DescriptionEditView.java

```diff
@@ -438,4 +444,12 @@ public void setAction(Action action) {
         this.action = action;
         isTranslationEdit = (action == TRANSLATE_CAPTION || action == TRANSLATE_DESCRIPTION);
     }
+
+    @Override
+    public void onLanguageDetectionSuccess(@NotNull String languageCode) {
+        if (!languageCode.equals("und") && !languageCode.equals(pageSummaryForEdit.getLang())) {
```

Comment:
The condition of the language being `"und"` can probably be moved into the `FirebaseLanguageDetector`, and it doesn't have to call the `success` callback in that case.

Reply:
@dbrant the service returns "und" [undetermined] when it is not able to determine with confidence. So we will need it here..